src/windows/com.rs

```diff
@@ -32,7 +32,17 @@ pub struct COMPort {
     port_name: Option<String>,
 }
 
+// The Windows [`HANDLE`] type is considered safe according to the standard library.
+// See explanation on [`OwnedHandle`] in stdlib for more information.
```

Comment:
Looking at https://doc.rust-lang.org/std/os/windows/raw/type.HANDLE.html or https://doc.rust-lang.org/std/os/windows/io/struct.OwnedHandle.html, I don't see the documentation you are mentioning at al first glance. Where is the statement made you are referring to? In [this comment in `handle.rs`](https://doc.rust-lang.org/src/std/os/windows/io/handle.rs.html#111)

Reply:
Yes, I'm referring to the comment in [`handle.rs`](https://doc.rust-lang.org/src/std/os/windows/io/handle.rs.html#111-117) that you linked.
I'm not sure how to link it to an `impl` block so I figured the definition of `OwnedHandle` might be the closest to that comment (50+ lines away)

The link to the std source https://doc.rust-lang.org/src/std/os/windows/io/handle.rs.html#111-117 doesn't seem to be a permanent one so I was worried putting the doc link in might become invalid in the future.

I can probably replace it with a GitHub permalink like this: https://github.com/rust-lang/rust/blob/f4f0fafd0c7849e162eddbc69fa5fe82dbec28c7/library/std/src/os/windows/io/handle.rs#L111-L115
but that would be a very long line unfortunately.

Reply:
Alight. A permalink would be great then because a long line looks way more appealing to me than the risk of getting it broken at some point in time.

Reply:
Thanks for your feedback! I've now updated the patch with a permalink to the explanation.